roles/openshift_manage_node/tasks/config.yml

```diff
@@ -9,3 +9,21 @@
   until: node_schedulable is succeeded
   when: "'nodename' in openshift.node"
   delegate_to: "{{ openshift_master_host }}"
+
+- name: Wait for sync DS to set annotations on all nodes
```

Comment:
Run sync DS check again after all node CSRs have been approved and node configs have landed